repositories.yaml

```diff
@@ -366,8 +366,6 @@ applications:
     deprecated: true
     notification_emails:
       - dthorn@mozilla.com
-      - mdroettboom@mozilla.com
```

Comment:
For deprecated apps, doesn't seem worth adding new members to notification list.